src/test/resources/ays_ui_elements_tr.properties

```
@@ -0,0 +1,3 @@
+emailAddress_errorMessage=Geçersiz e-posta adresi.
+password_errorMessage=Bu alan en az 6 karakter olmalıdır.
+header=Hoş Geldiniz
```

`.properties` uzantılı dosyalarda standart olabilmesi adına kelimeler `.` karakteri ile ayırılır. Ek olarak uzun vadeli düşündüğümüz durumda aşağıdaki dosya da şu şekilde yazılabilir. Buranın standartları ile ilgili de bir doküman hazırlanabilir. `<module>.<category>.<key>=<value>`

```suggestion
login.validation_error.email_address=Geçersiz e-posta adresi.
login.validation_error.password=Bu alan en az 6 karakter olmalıdır.
login.header.welcome=Hoş Geldiniz

```